src/lib/components/molecules/option-picker/index.jsx

```diff
@@ -0,0 +1,84 @@
+import { useState, useCallback } from "preact/hooks"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+import defaultStyles from "./style.module.css"
+
+/** @typedef {  "vertical" | "horizontal" } OptionLayoutDirection */
+
+export function OptionPicker({
+  title,
+  options,
+  layoutDirection = "horizontal",
+  onSelect,
+  styles,
+}) {
+  styles = mergeStyles(defaultStyles, styles)
+
+  const [selectedIndex, setSelectedIndex] = useState(0)
```

Comment:
Question: do we want the first option to be selected by default, always? Maybe an `initialSelected` prop would help?